webapp/src/components/sidebar/ModelLibrary.vue

```diff
@@ -24,31 +23,32 @@
           </span>
           <div v-if="memoryStore.useIndexedDB" class="space-y-4">
             <button
-              v-for="[id, metadata] in memoryStore.models"
+              v-for="[id, metadata] in memoryStore.models.sort((a : ModelMetadata, b : ModelMetadata) => sortModels(a, b))"
```

Comment:
I think there is no need for that, because he already react at the changes in the memorystore when you delete a model, and the ModelLibrary has to be reload if you add or modify a model.

Reply:
I see perfect!